README.md

```diff
@@ -31,6 +31,21 @@ docker build -t ipfs-check .
 docker run -d ipfs-check
 ```
 
+## Running locally
+
+### Terminal 1
+```
+go build
+./ipfs-check # Note listening port.. output should say something like "listening on [::]:3333"
+```
+
+### Terminal 2
+```
+# feel free to use any other tool to serve the contents of the /web folder, but you must run `npm run build` first.
+(cd web && npm run build) && npx -y serve -l 3000 web
+# Then open http://localhost:3000?backendUrl=http://localhost:3333
```

Comment:
Related to my question below, is this necessary? Can't you just open up index.html in your browser (with the backendUrl set to be local, or deployed elsewhere) and you're good to go.

Reply:
Unfortunately, the telemetry.js file has to be built in order for things to load properly. We could solve this with properly exported esm similar to how @whizzzkid did in ipfs-geoip, but the work has not been done for that yet.

I could fix things to allow users to just load web/index.html and have things work without a build step but will need to add a no-op web/dist/telemetry.js. I can do that.

Our deployment of ipfs-check will still require the build step though

Reply:
Ah ok, you need to build a minified telemetry.js with all the included dependencies and such. Makes sense, doesn't seem reasonable to block on you doing more work here. At some point this whole tool probably merges with pl-diagnose anyway #24 and has lots of fancy JS build tooling anyhow.

Reply:
Updated `web/dist/telemetry.js` to commit with an empty script file (comment only) and addressed some updates in the readme to be more accurate.